app/aimodels/bertopic/ai_services/basic_inference.py

```diff
@@ -234,9 +234,9 @@ def build_topic_model(self, documents_text_list, embeddings, num_topics, seed_to
             data_test = data_test[data_test['y_pred'] < 2]
             documents_text_list = list(data_test['message'])
 
-        hdbscan_model = hdbscan.HDBSCAN(min_cluster_size=10, min_samples=10,
+        hdbscan_model = hdbscan.HDBSCAN(min_cluster_size=2, min_samples=2,
```

Comment:
@erob123 reason for this change? min_cluster_size and min_samples came from Courtland's experimentation

Reply:
@erob123 I resolved this in https://github.com/MIT-AI-Accelerator/c3po-model-server/pull/39